models/api/flask_api.py

```diff
@@ -0,0 +1,141 @@
+import io
+
+from flask import Flask, jsonify,send_file, make_response
+from flask_cors import CORS, cross_origin
+import sys
+
+
+sys.path.insert(1, 'C:/Users/razan/Desktop/Kandidatarbetet/datx02-05/models')
+from visualizers.models_visual_training_comparator import Models_comparator
+import ast
+import MLmodels.DecisionTree as DecisionTree
+import MLmodels.ElasticNets as ElasticNet
+import MLmodels.Lasso as Lasso
+import MLmodels.NeuralNetwork as NeuralNetwork
+import MLmodels.RandomForest as RandomForest
+import MLmodels.Ridge as Ridge
+import numpy as np
+
+app = Flask(__name__)
+cors = CORS(app)
+app.config['CORS_HEADERS'] = 'Content-Type'
+
+last_used_model_name = ""
+last_used_training_set = ""
+
+@app.route("/")
+def index():
+    return 'Hello from  Models!'
+
+
+@app.route("/models")
+def model_names():
+    res = [
+        ["Lasso Model", "Lasso", "https://upload.wikimedia.org/wikipedia/commons/thumb/3/3a/Linear_regression.svg/1024px-Linear_regression.svg.png"],
+        ["Ridge Model", "Ridge", "https://upload.wikimedia.org/wikipedia/en/e/ed/First_order_hsked.svg"],
+        ["Elastic Net Model", "ElasticNet", "https://www.researchgate.net/profile/Bart_Hobijn2/publication/46566214/figure/fig1/AS:652216872488960@1532512028126/Okuns-law-before-and-during-the-2007-recession.png"],
+        ["Decision Tree Model", "DecisionTree", "https://upload.wikimedia.org/wikipedia/commons/f/ff/Decision_tree_model.png"],
+        ["Random Forest Model", "RandomForest", "https://upload.wikimedia.org/wikipedia/commons/c/c7/Randomforests_ensemble.gif"],
+        ["Neural Network Model", "NeuralNetwork", "https://live.staticflickr.com/8435/7880912598_389d98a505_b.jpg"]
```

Comment:
Some of these images might need to be replaced with images from public domain (Licensing issues, specifically elasticnet and neuralnet)

Reply:
all of them now are gotten from google with filter (får användas)